deploy/blobfuse2/config.yaml

```diff
@@ -0,0 +1,23 @@
+allow-other: true
+
+logging:
+  type: base
+  level: log_debug
+
+components:
+  - libfuse
+  - file_cache
+  - attr_cache
+  - azstorage
+
+libfuse:
+  attribute-expiration-sec: 120
+  entry-expiration-sec: 120
+  negative-entry-expiration-sec: 240
+
+file_cache:
```

Comment:
there was `path` parameter under `file_cache`, I have removed it since it could be replaced by `--tmp-path` in blobfuse2 mount option.